sources/packages/web/src/components/aest/AppealRequestsApprovalForm.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+  <appeal-requests-form
+    :studentAppealRequests="studentAppealRequests"
+    :readOnly="true"
+  >
+    <template #approval-form="{ approval }">
```

Comment:
I didn't get this part. I may need to sync up to understand this.

Reply:
Talked on the call and clarified 😉